lib/rules/sort-flags.ts

```diff
@@ -0,0 +1,181 @@
+import type { Literal } from "estree"
+import type { RegExpVisitor } from "regexpp/visitor"
+import type { RegExpContext } from "../utils"
+import { createRule, defineRegexpVisitor } from "../utils"
+
+export default createRule("sort-flags", {
+    meta: {
+        docs: {
+            description: "require the regex flags to be sorted",
```

Comment:
I have to correct my English here...

```suggestion
            description: "require regex flags to be sorted",
```